src/main/java/com/cglx/Calculator.java

```diff
@@ -28,12 +28,10 @@ public int substract(int a, int b)
 
     public int square(int a)
     {
-        int result = a * a;
-        return result;
+        return a * a;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "result". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-01.ahprod.adhaven.com:9990/coding_rules#rule_key=squid%3AS1488)